src/utils/twitch.js

```diff
@@ -133,6 +137,20 @@ export default {
     });
   },
 
+  async getCurrentProfile() {
+    if (currentProfile != null) {
+      return currentProfile;
+    }
+
+    try {
+      const res = await twitchAPI.get(`users?login=${currentUser.displayName}`);
```

Comment:
1. using a display name as the login is not reliable. they are distinctly different fields, and some users exist where `login != displayName.toLowerCase()`
2. using a login is brittle, since users can change their name. always use a twitch user id for resolution.

additionally, should we be using gql for this since it's onsite? twitch api v5 is deprecated